cmd/apiserver/server.go

```diff
@@ -60,12 +57,80 @@ func (s *server) createCalculationBulk(c *gin.Context) {
 	}
 
 	if err := s.client.Create(s.ctx, bulk); err != nil {
-		responseError(c, "couldn't create calculation", err)
+		responseError(c, "couldn't create calculation bulk", err)
 	} else {
 		c.JSON(http.StatusOK, gin.H{"data": bulk})
 	}
 }
 
+func (s *server) createWorkerPool(c *gin.Context) {
+	workerPoolName := c.Param("id")
```

Comment:
Since it's a POST request, we should get the worker pool name from a parm in the body.